supreme_court_predictions/statistics/descriptives.py

```diff
@@ -49,7 +49,8 @@ def __init__(self, downloaded_clean_corpus, save_data=True):
         else:
             print("Cleaned corpus already downloaded")
 
-    def fix_indices(self, main, sub):
+    @staticmethod
+    def fix_indices(main, sub):
```

Comment:
Made method static since it does not reference the `self` variable at all. 

What is a static method: [Link](https://www.digitalocean.com/community/tutorials/python-static-method)